user-manual/latest.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Latest User Manual
+redirect_to: https://accumulo.apache.org/1.7/accumulo_user_manual.html
+---
+
+This page redirects to latest Accumulo Manual
```

Comment:
If I understand the redirect plugin correctly, this uses meta-refresh. The text of the page should be useful, like a link to the page, if redirects are disabled in the user's browser, or the redirect is interrupted.